scripts/ec_transfer_records.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# Zenodo-RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""Script to transfer communities and records to new parent.
+
+Usage:
+
+.. code-block:: python
+
+    _to = "ec_community"
+    _from = ["one_eu_project", "another_maybe"]
+    
+    # option 1 - move communities to new parent (including records)
+    transfer_communities(identity, _from, _to, set_default=False)
+    
+    # option 2 - move records in bulk
+    records = ["1234561", "1234321"]
+    transfer_records(identity, records, _to, set_default=False)
+
+"""
+
+from invenio_communities.proxies import current_communities
+from invenio_rdm_records.proxies import current_rdm_records
+from invenio_records_resources.services.uow import UnitOfWork
+from invenio_search.engine import dsl
+from werkzeug.local import LocalProxy
+
+community_service = LocalProxy(lambda: current_communities.service)
+records_communities_service = LocalProxy(
+    lambda: current_rdm_records.record_communities_service
+)
+records_service = LocalProxy(lambda: current_rdm_records.records_service)
```

Comment:
Question: Do you need the `LocalProxy`? it's already a proxy (We might miss something ....)

Reply:
Without using `LocalProxy`, the value will be resolved as soon as the file is loaded and, in that moment, the app context from flask may not exist yet. 

This way, the value will be resolved on demand (when the app context already exists theoretically).